app/styles/spacing.scss

```diff
@@ -0,0 +1,72 @@
+// Spacing definitions
+
+$space-2xs: 0.25rem; // 4px
+$space-xs: 0.5rem; // 8px
+$space-s: 0.75rem; // 12px
+$space-m: 1rem; // 16px
+$space-l: 1.25rem; // 20px
+$space-xl: 1.5rem; // 24px
+$space-2xl: 2rem; // 32px
+$space-3xl: 2.5rem; // 40px
+$space-4xl: 3rem; // 48px
```

Comment:
We miiiight need to move this into vars actually if we want to use them in module.scss files?

Reply:
can do